.github/workflows/ci.yml

```
@@ -28,17 +28,16 @@ jobs:
       run: |
         luarocks install luafilesystem
         luarocks install luacov
-        luarocks install luacov-coveralls --server=http://rocks.moonscript.org/dev
+        git clone https://github.com/alerque/luacov-coveralls.git --branch fix-actions-service-name
```

This close is correct, but Luarocks then procedes to use the URL in the spec to download what it wants! I posted a rockspec just for testing this that downloads from the right branch:

```suggestion
        wget https://gist.githubusercontent.com/alerque/09ec33446e69f60243b80c28d4b44aec/raw/21574b22ef3c2511a578f74c193622d1fdff2bd2/luacov-coveralls-TEST-0.rockspec
```